sentry-delayed_job/spec/sentry/delayed_job_spec.rb

```diff
@@ -234,6 +237,93 @@ def perform
         expect(event[:contexts][:"Active-Job"][:job_class]).to eq("FailedJob")
       end
     end
+
+    context "when tracing is enabled" do
+      before do
+        perform_basic_setup do |config|
+          config.traces_sample_rate = 1.0
+        end
+      end
+  
+      it "records transaction" do
+        ReportingJob.perform_later
+
+        enqueued_job = Delayed::Backend::ActiveRecord::Job.last
+        enqueued_job.invoke_job
+
+        expect(transport.events.count).to eq(3)
+        transaction = transport.events.last
+  
+        expect(transaction.transaction).to eq("ReportingJob")
+        expect(transaction.contexts.dig(:trace, :trace_id)).to be_a(String)
+        expect(transaction.contexts.dig(:trace, :span_id)).to be_a(String)
+        expect(transaction.contexts.dig(:trace, :status)).to eq("ok")
+      end
+  
+      it "records transaction with exception" do
+        FailedJob.perform_later
+        enqueued_job = Delayed::Backend::ActiveRecord::Job.last
+        begin
+          enqueued_job.invoke_job
+        rescue ZeroDivisionError
+          nil
+        end
+  
+        expect(transport.events.count).to eq(4)
```

Comment:
I have no idea why the event count is 2 larger than without activejob. Is this a problem? If so, hints as to why are appreciated.